libraries/RC_Channel/RC_Channels.cpp

```diff
@@ -89,6 +90,20 @@ bool RC_Channels::read_input(void)
         success |= channel(i)->update();
     }
 
+    // check if clear overrides by RC is allowed by RC_OPTIONS and any change in RC input during RC overrides
+    if (rc().clear_overrides_by_rc() && has_active_overrides() &&
+        (channel(AP::rcmap()->roll() - 1)->rc_in_changed ||
```

Comment:
Why don't we just check to see if any of them are outside their deadzones - with the exception of throttle, which we eithre don't check or *do* check for a change...

... and maybe just have a `uint16_t last_throttle` rather than the `rc_in_changed` infrastructure.

We should probably also very that RC input is actually valid before checking any channel value!